### ISSUE TITLE
Focus highlight not quite enough contrast

### ISSUE THREAD

**Issue by bruce-usab** [uswds/uswds#3242]
I expected to find that the default focus highlight (cyan) would provide good contrast (3:1) against both the button background color and white.

From [pull request 2604](http://github.com/uswds/uswds/pull/2604) the work around seems to be to put a white margin between the button and the focus.  With small tweaks to the colors, this would not be necessary.  Please compare:

[Current button/page/focus colors](http://webaim.org/resources/linkcontrastchecker/?fcolor=0068B4&bcolor=FFFFFF&lcolor=009AFA)
[Suggested button/page/focus colors](http://webaim.org/resources/linkcontrastchecker/?fcolor=004488&bcolor=FFFFFF&lcolor=0097F5)

This relates to [WCAG 2.1 SC 1.4.11](http://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html).

The benefit would be a default palette that provides better contrast for all users, but especially folks with low vision.

The challenge, of course, is that any tweaks to the hex values has to be be tested so as not to mess up the use of tokens.

**Comment by bruce-usab** [comment]
@thisisdano no problem to close, but is it OBE or already done or more a won't fix?  My apologies for not being able to tell from the PR.

**Comment by thisisdano** [comment]
Sorry, that's my fault. I should have given more context.

In this case, we're considering it a `wontfix`, as the focus highlight has appropriate contrast against the background, and the button has appropriate contrast against the background, and the margin between them serves as a second border, as in this example from [the W3 contrast page](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) — the last example is the correct one and similar to our current case:

![radio-custom](https://user-images.githubusercontent.com/11464021/167453960-a96f7751-5150-41fb-8c07-b090e28e0a34.png)

This allows our focus to work reliably with all button colors and variants that, themselves, have proper contrast.
